### ISSUE TITLE
[BUG] Can't launch games

### ISSUE THREAD

**Issue by mnhyim** [Dummerle/Rare#251]
**Describe the bug**
Games wont launch.

**To Reproduce**
1. Open Rare
2. Launch a game

**Expected behavior**
Games should be running normally.

**Screenshots**
![image](https://user-images.githubusercontent.com/34624746/196348198-c413c7ef-feaf-404a-af9a-1f11c7e78187.png)

**Desktop (please complete the following information):**
- OS: Windows 10
- Version: 1.9.1
- Installation: pip
- Python version: 3.10.8

**Additional context**

**Error message**
[GameUtils] ERROR: 41869934302e4b8cafac2d3c0e7c293d: QLocalSocket::connectToServer: Invalid name
[GameUtils] INFO: Game exited with exit code: 1

**Comment by loathingKernel** [comment]
Please attach the logs of `Rare` and `Rare_Launcher` for the game at the time of the failure. You can open the folder through Rare's settings.

**Comment by mnhyim** [comment]
Okay, updated the Issues with the log files.

**Comment by loathingKernel** [comment]
We also need the `Rare_Launcher_<appid>` log from the same directory

**Comment by mnhyim** [comment]
Weird, there's no `Rare_Launcher_<appid>` logs in that directory.

**Comment by loathingKernel** [comment]
<appid> is a placeholder for the internal name of the game, in your previous trial it should be `Rare_Launcher_41869934302e4b8cafac2d3c0e7c293d_something`, but since I don't know how and what games you have tried, you have to infer that yourself.

**Comment by mnhyim** [comment]
yeah I cant find any other log files except for `Rare_22-10-18--18-49.log`. I tried launching the games using Legendary and it worked normally, so I think there's something wrong with my Rare installation.

**Comment by loathingKernel** [comment]
It's the pip method, it had a bug on windows. The big has been fixed, but the Readme has to be updated to not recommend pip on windows anyways. We have better solutions for windows, like the zip or the msi.

**Comment by Dummerle** [comment]
Does it work in the new release? Then the issue can be closed
